web_src/js/markup/math.js

```diff
@@ -0,0 +1,42 @@
+function displayError(el, err) {
+  let target = el;
+  if (el.classList.contains('is-loading')) {
+    // assume no pre
+    el.classList.remove('is-loading');
+  } else {
+    target = el.closest('pre');
+    target.classList.remove('is-loading');
+  }
```

Comment:
I've seen the block `return el.classList.contains('is-loading') ? el : el.closest('pre')` now at least twice in this file.
Time for a function.
This will reduce this one and the other block to a single line.